book/environment.md

```diff
@@ -33,15 +33,15 @@ Using the `let-env` command is the most straightforward method
 > let-env FOO = 'BAR'
 ```
 
-'let-env' is similar to the **export** command in bash.
+'let-env' is similar to the **export** command in Bash.
 
-So if you want to extend the `PATH` variable for example, you could do that as follows.
+So, if you want to extend the Windows `Path` variable, for example, you could do that as follows.
 
 ```
-let-env PATH = ($env.PATH | prepend '/path/you/want/to/add')
+let-env Path = ($env.Path | prepend '\path\you\want\to\add')
```

Comment:
and you might want to have a drive letter in the path too like `c:\path\you\want\to\add`